packages/core/src/odata-common/request-builder/request-builder-base.ts

```diff
@@ -132,6 +132,17 @@ export abstract class MethodRequestBuilder<
     return this;
   }
 
+  /**
+   * Append additional path to the generated path. Typically, this can be used for querying navigation properties of an entity.
+   * When this function is used, the "execute" function of the request builder is disabled and use "executeRaw" instead.
+   * @param path Path to be appended.
+   * @returns The request builder itself without "execute" function, to facilitate method chaining.
+   */
+  appendPath(path: string): Omit<this, 'execute'> {
```

Comment:
We changed the naming in the builder to a `addXYZ` and `setXYZ` which adds or sets something. This one is currently a `add` so perhaps follow this pattern @marikaner?

Reply:
I personally would use `append` as a verb, because it clearly means you "add A to the **end** of B"